src/types/webhooks/execute_webhook.ts

```diff
@@ -0,0 +1,25 @@
+import { Embed } from "../embeds/embed.ts";
+import { AllowedMentions } from "../messages/allowed_mentions.ts";
+import { SnakeCaseProps } from "../util.ts";
+
+export interface ExecuteWebhook {
+  /** Waits for server confirmation of message send before response, and returns the created message body (defaults to `false`; when `false` a message that is not saved does not return an error) */
+  wait?: boolean;
```

Comment:
`wait` is a querystring parameter, not a JSON paramater.

Reply:
We accept it as a param